WebHook/WebHookFunction.cs

```diff
@@ -146,7 +146,7 @@ public static class WebHookFunction
             var relevantFiles = hook.commits.SelectMany(x => x.added)
                 .Concat(hook.commits.SelectMany(x => x.modified));
             var imageFiles = relevantFiles.Where(file => KnownImgPatterns.ImgExtensions.Any(extension => file.ToLower().EndsWith(extension, StringComparison.Ordinal)));
-            var configFile = relevantFiles.Where(file => file.ToLower() == ".imgbotconfig");
+            var configFile = relevantFiles.Where(file => file.ToLower() == ".imgbotconfig" || file.ToLower() == ".imgbotconfig.json");
```

Comment:
We probably want to change `ToLower` to `ToLowerInvariant` here.

Reply:
should that be updated for both the comparisons?

Reply:
Yup